cmd/query/app/static_handler.go

```diff
@@ -72,6 +74,12 @@ type StaticAssetsHandlerOptions struct {
 	Logger       *zap.Logger
 }
 
+// UIConfigOptions define options for injecting config in index.html in loadAndEnrichIndexHTML
+type UIConfigOptions struct {
```

Comment:
* "options" typically refers to inputs, this is used as output
*  does not need to be public

```suggestion
type loadedConfig struct {
```